src/main/java/org/folio/rest/converter/RMAPIConfigurationConverter.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.rest.converter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.config.RMAPIConfiguration;
+import org.folio.rest.jaxrs.model.EholdingsConfigurationPutApplicationVndApiJsonImpl;
+
+/**
+ * Converts internal RMAPIConfiguration object into object that will be returned in response
+ */
+public class RMAPIConfigurationConverter {
```

Comment:
@andrii-paias : What do you think of renaming this class to RMAPIConfigurationSerializer instead of Converter? Or is Converter better understandable in Java space?

Reply:
My reasoning was that serializer normally means a class that serializes an object to json string, or has something to do with java Serializable interface, so I named it converter. But I don't have a strong opinion about it, if this will make it consistent with other modules we can rename it to serializer.

Reply:
I do not have a strong opinion either. Let's leave it the way it is for now.